camus-etl-kafka/src/main/java/com/linkedin/camus/etl/kafka/mapred/EtlInputFormat.java

```diff
@@ -416,8 +405,10 @@ public int compare(CamusRequest r1, CamusRequest r2) {
       if (request.getEarliestOffset() > request.getOffset() || request.getOffset() > request.getLastOffset()) {
         if (request.getEarliestOffset() > request.getOffset()) {
           log.error("The earliest offset was found to be more than the current offset: " + request);
+          EmailClient.sendEmail("The earliest offset was found to be more than the current offset: " + request);
```

Comment:
if `move_to_earliest_offset=false`, then the `else` block in line 426 will be executed, the job will fail and ops will get the alert.

if `move_to_earliest_offset` is set to true, then it means whoever runs the job expects this type of situation, and shouldn't be alerted if this happens.